app/views/admin/enterprises/form/_connected_apps.html.haml

```diff
@@ -1,5 +1,4 @@
-- enterprise ||= f.object
-#connected-app-discover-regen
+%div{ id: "connected-app-discover-regen", class: "enterprise_#{enterprise.id}" }
```

Comment:
The background job used the form id in its selector for updates but now we are outside the form, it's better to provide the selector ourselves here. Having a variable class name means that we need to use the explicit `%div` syntax with attribute hash.

Reply:
I would have expected this to work also:
```
#connected-app-discover-regen{ class: "enterprise_#{enterprise.id}" }
```
But maybe it doesn't. Either way, it really doesn't matter.

Reply:
I tried that and somehow it didn't work. :shrug: